### DISCUSSION TITLE
DNDBeyond, Beyond20 & Dice-stats

### DISCUSSION THREAD

**Discussion by joebags99** [jacobwojoski/dice-stats#69] [Q&A] [answered]
Will the module work with dice rolled in DNDBeyond and brought over to Foundry via Beyond20?

I couldn't find anything in the documentation so I figured I would ask it here. Thanks!

**Comment by jacobwojoski** [comment]
Honestly don't know. I only play using pf2e with minor testing on D&D for some module issues. 

It depends how it transfers roll info. 
Inside the chat message object foundry has a data value that holds roll info along with if the chat message was a roll. So if the imported chat message from beyond 20 doesn't populate those fields then the mod wont work. if it does populate those fields then it should be fine.
 

Chat Message Object layout (Default to foundry I didn't make this object, Guessing the pf2e system might edit it a bit though. This section of data should be constant between systems.)
![image](https://github.com/jacobwojoski/dice-stats/assets/124112219/62f2dcf3-1238-4cd3-9897-2473430b54ad)

**Reply by joebags99** [reply, marked as answer]
So I discovered the issue! If anyone runs into this issue in the future all they need to do is turn off "Use D&DBeyond's Digital Dice" through the Beyond20 extension. When I turn this feature off, the module works as intended. 

It seems like since it when D&DBeyond rolls the dice, it does not import the necessary data. 

Thanks for the info! It helped me solve the problem :)

**Reply by jacobwojoski** [reply]
Awesome! Glad I could help!
